Automattic/Simplecons
# Optimise SVG Code

This is the first PR to optimise the SVG code of Simplecons. 

The goal of optimisation as a whole is to:
- Make sure all icons only contain a single path.
- Make sure all icons have their path decimal precision set to 3 decimal places.

In this PR, I am removing all transforms, rotates, circles or polygons from the code so that only `<path>` tags exist within the main `<g>` tags of some of the icons. All modified icons have their paths decimal precision set to 3 and contain only a single path.

The icons affected by this change are:

**24px**
- add-outline
- add
- analytics
- archive
- arrow-left
- arrow-top-right
- checklist
- chevron-right
- cross-outline
- cross
- hide-keyboard
- info
- new-note (icon has been shifted to the right by 1px for better center alignment)
- published
- restore
- search
- trash
- unpublished
- untagged

**16px**
- arrow
- checklist
- chevron-right
- cross-outline
- cross
- published
- search

### To Test
- Verify that only `path` tags now exist inside the main `g` container in all icons.
- Verify that the icons listed above have not changed visually compared with their non-optimsed versions.
